SelfCarState[] WAITING_PICKUP
SelfCarState[] RUNNING
SelfCarState[] WAITING_GOTO_GW    # 等待去往地勤作业点
SelfCarState[] WAITING_GOTO_AW    # 等待去往航空作业点
SelfCarState[] WAITING_UAV_WORKING